node/CMakeLists.txt

```diff
@@ -29,6 +29,14 @@ set(SILKWORM_INTERFACE_SRC
         ${SILKWORM_MAIN_DIR}/interfaces/p2psentry/sentry.grpc.pb.h
         ${SILKWORM_MAIN_DIR}/interfaces/p2psentry/sentry.pb.cc
         ${SILKWORM_MAIN_DIR}/interfaces/p2psentry/sentry.pb.h
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/ethbackend.grpc.pb.cc
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/ethbackend.grpc.pb.h
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/ethbackend.pb.cc
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/ethbackend.pb.h
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/kv.grpc.pb.cc
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/kv.grpc.pb.h
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/kv.pb.cc
+        ${SILKWORM_MAIN_DIR}/interfaces/remote/kv.pb.h
```

Comment:
Why did you choose the name "remote" for the directory that contains ethbackend and kv generated code?

Reply:
Because it just matches the same name used in the [interfaces repo](https://github.com/ledgerwatch/interfaces). It seems useful to maintain the same naming for coherence, as already done for `p2psentry` and `types`.